hathor/metrics.py

```diff
@@ -159,11 +159,17 @@ def start(self) -> None:
         self.is_running = True
         self._lc_collect_data.start(self.collect_data_interval, now=False)
 
+        if self.websocket_factory:
```

Comment:
Why is it being initialized by the metrics singleton? I'd say the best place to start/stop is in the `HathorManager`.

Reply:
Fixed. Can you take another look?